src/Tests/Microsoft.DotNet.ApiCompatibility.Tests/Rules/CannotAddOrRemoveVirtualKeywordTests.cs

```diff
@@ -203,5 +205,41 @@ public class First {
             };
             Assert.Equal(expected, differences);
         }
+
+        [Fact]
+        public static void EnsureDiagnosticWhenAddingSealedToInterfaceMember()
+        {
+            if (IsNetFramework)
+            {
+                return;
+            }
```

Comment:
The test project already compiles separately for .NET Framework: https://github.com/dotnet/sdk/blob/cae591210531dc36615f9cd78fc79af9834e1084/src/Tests/Microsoft.DotNet.ApiCompatibility.Tests/Microsoft.DotNet.ApiCompatibility.Tests.csproj#L4

So you can just add a preprocessor symbol and exclude the whole method from the .NETFramework build:

```C#
// .NET Framework doesn't support default interface methods.
#if !NETFRAMEWORK
[Fact]
public static void EnsureDiagnosticWhenAddingSealedToInterfaceMember()
{
...
}
#endif
```